opendcs-web-client/src/main/webapp/WEB-INF/web.xml

```diff
@@ -27,6 +27,10 @@
         <welcome-file>default.htm</welcome-file>
         <welcome-file>default.jsp</welcome-file>
     </welcome-file-list>
+    <context-param>
+        <param-name>api_base_url</param-name>
+        <param-value>http://opendcsapi:8080/odcsapi</param-value>
```

Comment:
For the moment I think it's reasonable to assume same host and "odcsapi" as the context. We can address the possible flexibility later. This is forcing HTTP instead of HTTPS and a specific domain name. 

Thus will make testing by others somewhat difficult in several situations.

Reply:
Sounds good - I've addressed this by going back to base path and assuming same host.